tests/test-distribution/src/test/resources/session-store-hazelcast-remote.ini

```diff
@@ -0,0 +1,16 @@
+# ---------------------------------------
+# Module: session-store-hazelcast-remote
+# Enables session data store in a remote Hazelcast Map
+# ---------------------------------------
+jetty.webapp.addSystemClasses+=,org.eclipse.jetty.hazelcast.
+jetty.webapp.addServerClasses+=,-org.eclipse.jetty.hazelcast.
+
+--module=session-store-hazelcast-remote
+
+#jetty.session.hazelcast.mapName=jetty-distributed-session-map
+jetty.session.hazelcast.hazelcastInstanceName=JETTY_DISTRIBUTED_SESSION_INSTANCE
+jetty.session.hazelcast.onlyClient=false
```

Comment:
@janbartel this is the reason this whole effort got started.

If `jetty.session.hazelcast.onlyClient=false` then we have classloader issues from within hazelcast.

Reply:
Can you clarify: if using hazelcast-embedded module (where onlyClient is false by default) does this problem occur or not?

Reply:
OK, I see what is happening. 

The test is setting the SessionSerializer in the _hazelcast_ config file so it is _hazelcast_ that is trying to load the SessionSerializer from the webapp classloader.  The way we recommend and have tested with is to let jetty set up the SessionSerializer in the HazelcastSessionDataStoreFactory, in which case there is no classloading issue.

I guess there is no harm in adding these 2 lines to the .mod file to support the usecase illustrated by the test.

Reply:
Yes, I agree, these 2 lines to adjust the webapp classloader should be in the mod file.

Reply:
fixed